presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/BigintDecoder.java

```diff
@@ -36,7 +45,7 @@ else if (value instanceof Number) {
             BIGINT.writeLong(output, ((Number) value).longValue());
         }
         else {
-            throw new PrestoException(TYPE_MISMATCH, "Expected a numeric value for BIGINT field");
+            throw new PrestoException(TYPE_MISMATCH, format("Expected a numeric value for field '%s' of type BIGINT: %s", path, value));
```

Comment:
i think `value.getClass()` is more helpful than `value.toString()`

for example you may get
```
Expected a numeric value for field 'my_col' of type BIGINT: 42
```

and this would be not much more useful than the message without "42"

i saw your comment  in https://github.com/prestosql/presto/pull/3321#discussion_r403103442
i still think we should include both: value.toString and value type.
If not for end-users, this be useful for us to provide help when they show up with an error.

Reply:
ok, fair enough.